elyra/metadata/tests/test_utils.py

```diff
@@ -22,27 +22,65 @@
 
 valid_metadata_json = {
     'schema_name': 'test',
-    'display_name': 'valid runtime',
+    'display_name': 'valid metadata instance',
     'metadata': {
-        'api_endpoint': 'http://localhost:31823/v1/models?version=2017-02-13',
-        'foo': 8
+        'uri_test': 'http://localhost:31823/v1/models?version=2017-02-13',
+        'number_range_test': 8,
+        'required_test': "required_value"
     }
 }
 
 another_metadata_json = {
     'schema_name': 'test',
-    'name': 'another_foo',
-    'display_name': 'Another Runtime (2)',
+    'name': 'another_instance',
+    'display_name': 'Another Metadata Instance (2)',
     'metadata': {
-        'api_endpoint': 'http://localhost:8081/'
+        'uri_test': 'http://localhost:8081/',
+        'required_test': "required_value"
     }
 }
 
 invalid_metadata_json = {
     'schema_name': 'test',
-    'display_name': 'Invalid Runtime',
+    'display_name': 'Invalid Metadat Instance',
```

Comment:
small typo..lol..had to do a double take since this was an invalid one and needed to check if that was what you were checking for

Reply:
:smile: Sorry about that.  That must have been a little adventure hassle.  Thannks for the catch.